# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1279634460 25200
#      Tue Jul 20 07:01:00 2010 -0700
# Node ID b988a45b60052f3f4c07ddf424b359093f1004a7
# Parent  bdde9c59f77ae2455e6a1aa6957aea43b4500ae0
ChangeLog entry for last checkin.

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,13 @@
+2010-07-20  Rik <octave@nomad.inbox5.com>
+
+	* DLD-FUNCTIONS/typecast.cc (typecast): Combine two @seealso macro
+	uses to one.
+
 2010-07-20  Jaroslav Hajek  <highegg@gmail.com>
 
 	* DLD-FUNCTIONS/dlmread.cc (Fdlmread): Use octave_idx_type instead of
 	unsigned long. Over-allocate by a factor 2 to reduce copying.
 	Attempt to estimate number of columns from first line.
 	Hoist tmp_stream out of the loop and reset it for each substring.
 
 2010-07-17  Rik <octave@nomad.inbox5.com>
